releasenotes/notes/2.0/prepare-2.0.0-bab067ae93d40bb1.yaml

```
@@ -0,0 +1,35 @@
+---
+prelude: >
+    We are pleased to release Qiskit v2.0.0, with new features that improve performance and its capabilities.
+    The feature highlights of Qiskit v2.0.0 include:
+
+      * The introduction of a C API for building and interacting with
+        :class:`.SparseObservable` objects. This first 
+        compiled language interface is an experimental feature that will 
+        be fine-tuned and will grow in scope throughout the v2.x release series.
+
+      * The ability to create ``stretch`` durations for :class:`.Delay` instructions,  
+        which enable expressing relationships between timing-aware instructions.  
+        The new :class:`~.expr.Stretch` type extends the existing classical expression system,  
+        allowing design intent to be captured at circuit construction time and resolved at compile time.
+        See the `OpenQASM documentation <https://openqasm.com/language/delays.html#duration-and-stretch-types>`__ for details.
+
+      * Support for synthesis using fractional basis gates,
+        significantly reducing gate counts in elegible circuits (see `blog post <https://www.ibm.com/quantum/blog/fractional-gates>`__ for more details).
```

We technically supported this before it just wasn't very good support since it always used a fixed angle for the rzz gate. I also would remove the blog post link a bit to decouple it from IBM's hardware announcement. The improvement here was motivated by the hardware exposing rzz on some Heron's but it's impact is more broad than that. If want to include the link I'd couple it specifically to the hardware by adding a sentence to the end of suggestion below:

 "This improves the transpiler quality when running on IBM Heron systems that are using
fractional gates (see: `blog post <https://www.ibm.com/quantum/blog/fractional-gates>`__ for more details)."

```suggestion
      * Improved synthesis when a :class:`.Target` contains fractional two qubit basis gates with support
         for arbitrary angles such as :class:`.RZXGate` or :class:`.RZZGate` which significantly reducing gate counts in the synthesized circuits when using these
        gates.
```